Node e2e build failed on MacOS

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

**Kubernetes version** (use `kubectl version`):
master

**Environment**:
- **OS**:
MacOS
- **Kernel** (e.g. `uname -a`):
Darwin Klauss-MacBook-Pro.local 15.6.0 Darwin Kernel Version 15.6.0: Mon Aug 29 20:21:34 PDT 2016; root:xnu-3248.60.11~1/RELEASE_X86_64 x86_64

**What happened**:

Build failed:

```
klauss-mbp:kubernetes klaus$ make
+++ [0920 22:00:50] Generating bindata:
    /Users/klaus/Workspace/kubernetes/test/e2e/framework/gobindata_util.go
+++ [0920 22:00:50] Building the toolchain targets:
    k8s.io/kubernetes/hack/cmd/teststale
+++ [0920 22:00:50] Building go targets for darwin/amd64:
    cmd/kube-dns
    cmd/kube-proxy
    cmd/kube-apiserver
    cmd/kube-controller-manager
    cmd/kubelet
    cmd/kubemark
    cmd/hyperkube
    plugin/cmd/kube-scheduler
    cmd/kubectl
    cmd/gendocs
    cmd/genkubedocs
    cmd/genman
    cmd/genyaml
    cmd/mungedocs
    cmd/genswaggertypedocs
    cmd/linkcheck
    examples/k8petstore/web-server/src
    federation/cmd/genfeddocs
    vendor/github.com/onsi/ginkgo/ginkgo
    test/e2e/e2e.test
    vendor/github.com/onsi/ginkgo/ginkgo
    test/e2e_node/e2e_node.test
# k8s.io/kubernetes/test/e2e_node
test/e2e_node/container_manager_test.go:80: undefined: getPidsForProcess
test/e2e_node/container_manager_test.go:80: undefined: dockerProcessName
test/e2e_node/container_manager_test.go:80: undefined: dockerPidFile
test/e2e_node/container_manager_test.go:89: undefined: getPidsForProcess
test/e2e_node/container_manager_test.go:89: undefined: kubeletProcessName
test/e2e_node/container_manager_test.go:116: undefined: getPidsForProcess
test/e2e_node/container_manager_test.go:130: undefined: getPidsForProcess
test/e2e_node/container_manager_test.go:168: undefined: getPidsForProcess
test/e2e_node/container_manager_test.go:209: undefined: getPidsForProcess
make: *** [all] Error 1
```

**What you expected to happen**:
Successfully built

**How to reproduce it** (as minimally and precisely as possible):

```
$git checkout master
$git fetch upstream
$git merge upstream/master
$make
```

**Anything else do we need to know**:
N/A

similar issue after #32884 , refer to the following output. Is the #32884 used to remove node_e2e on Mac? I'll check it and try to fix this issue :).

```
Klauss-MacBook-Pro:kubernetes klaus$ git log --stat  | grep 32884
    Merge pull request #32884 from Random-Liu/separate-build-for-node-e2e
^C
Klauss-MacBook-Pro:kubernetes klaus$ make
+++ [0921 17:34:37] Generating bindata:
    /Users/klaus/Workspace/kubernetes/test/e2e/framework/gobindata_util.go
+++ [0921 17:34:37] Building the toolchain targets:
    k8s.io/kubernetes/hack/cmd/teststale
+++ [0921 17:34:37] Building go targets for darwin/amd64:
    cmd/kube-dns
    cmd/kube-proxy
    cmd/kube-apiserver
    cmd/kube-controller-manager
    cmd/kubelet
    cmd/kubemark
    cmd/hyperkube
    plugin/cmd/kube-scheduler
    cmd/kubectl
    cmd/gendocs
    cmd/genkubedocs
    cmd/genman
    cmd/genyaml
    cmd/mungedocs
    cmd/genswaggertypedocs
    cmd/linkcheck
    examples/k8petstore/web-server/src
    federation/cmd/genfeddocs
    vendor/github.com/onsi/ginkgo/ginkgo
    test/e2e/e2e.test
    vendor/github.com/onsi/ginkgo/ginkgo
    test/e2e_node/e2e_node.test
# k8s.io/kubernetes/test/e2e_node
test/e2e_node/container_manager_test.go:80: undefined: getPidsForProcess
test/e2e_node/container_manager_test.go:80: undefined: dockerProcessName
test/e2e_node/container_manager_test.go:80: undefined: dockerPidFile
test/e2e_node/container_manager_test.go:89: undefined: getPidsForProcess
test/e2e_node/container_manager_test.go:89: undefined: kubeletProcessName
test/e2e_node/container_manager_test.go:116: undefined: getPidsForProcess
test/e2e_node/container_manager_test.go:130: undefined: getPidsForProcess
test/e2e_node/container_manager_test.go:168: undefined: getPidsForProcess
test/e2e_node/container_manager_test.go:209: undefined: getPidsForProcess
make: *** [all] Error 1
```

--- a/test/e2e_node/container_manager_test.go
+++ b/test/e2e_node/container_manager_test.go
@@ -1,3 +1,5 @@
+// +build linux
+
 /*
 Copyright 2016 The Kubernetes Authors.
 

